Assets/HoloToolkit/Utilities/Scripts/HeadsUpDirectionIndicator.cs

```diff
@@ -15,7 +15,8 @@ namespace HoloToolkit.Unity
     public class HeadsUpDirectionIndicator : MonoBehaviour
     {
         [Tooltip("The object the direction indicator will point to.")]
-        public GameObject TargetObject { get; set; }
+        [SerializeField]
```

Comment:
I think keeping this a public property would be good in case people want to overwrite the TargetObject at runtime.  Also, if it's a property then you will not be able to assign the object in the inspector window nor use the Tooptip.